deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1985,6 +1986,12 @@ def registry_value_cud(root_key, registry_sub_key, log_monitor, arch=KEY_WOW64_6
         Set with all the checkers. Default `None`
     triggers_event : boolean, optional
         Boolean to determine if the event should be raised or not. Default `True`
+    triggers_event_add: boolean, optional
+        Boolean to determine if the added event should be raised. If triggers_event is false, this parameter is ignored
+    triggers_event_modified: boolean, optional
+        Boolean to determine if the modified event should be raised. If triggers_event is false, this parameter is ignored
```

Comment:
Don't exceed 120 characters per line. Also, correct the punctuation, there is a missing `.` in the three new descriptions.